### PR TITLE
generate SDK for new preview July version + updated unit tests

### PR DESCRIPTION
**Pull request by sotaneja**
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/10180

### CODE REVIEW

**Review by isra-fel — CHANGES_REQUESTED** [review]
The changes LGTM. Just please update the version and release notes in your csproj, and the version in `AssemblyInfo.cs` as well. Thanks

**Comment by sotaneja** [comment]
> The changes LGTM. Just please update the version and release notes in your csproj, and the version in `AssemblyInfo.cs` as well. Thanks

Thanks for the quick turnaround. I have applied this feedback.

File: sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Microsoft.Azure.Management.Maintenance.csproj [hunk 1]
```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Maintenance</PackageId>
     <Description>Microsoft Azure Maintenance Management Library</Description>
-    <Version>1.0.0</Version>
+    <Version>2.0.0</Version>
```
Comment (isra-fel) [inline sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Microsoft.Azure.Management.Maintenance.csproj:9]: We follow sementic versioning, so a major version update means there's going to be breaking changes for the user. Are you sure about this?

In normal case when there's some non-breaking feature addition, we usually update the minor version, i.e. `1.1.0`. If that suits you, don't forget the version in AssemblyInfo.cs

Comment (sotaneja) [reply sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Microsoft.Azure.Management.Maintenance.csproj]: I see ok I have updated only the minor version then

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**